src/utils/request/parseIsomorphicRequest.ts

```diff
@@ -15,7 +16,8 @@ export function parseIsomorphicRequest(
     url: request.url,
     method: request.method,
     body: parseBody(request.body, request.headers),
-    headers: request.headers,
+    credentials: request.credentials,
```

Comment:
We should still have the `same-origin` fallback to preserve the previous behavior. 

```suggestion
    credentials: request.credentials || 'same-origin',
```